.github/workflows/cpp.yml

```diff
@@ -55,23 +55,26 @@ jobs:
         addlicense -check .
 
   build:
+    name: build (${{ matrix.runtime }})
+
     runs-on: ubuntu-latest
 
     strategy:
       matrix:
```

Comment:
Could you add `fail-fast: false` under `strategy`? Without it, all jobs are being cancelled if one of them fails.
```
strategy:
  fail-fast: false
  matrix:
    ...
```

Reply:
done. thanks for the suggestion!